packages/firestore/src/util/types.ts

```diff
@@ -47,3 +47,18 @@ export function isSafeInteger(value: unknown): boolean {
     value >= Number.MIN_SAFE_INTEGER
   );
 }
+
+/** The subset of the Window interface used by the SDK. */
```

Comment:
Is `Window` universally available? I thought it was specific to the Browser? If so, minor nit: this is a subset of the browser's Window class. Same nit below.

Reply:
Fixed